### ISSUE TITLE
[BUG] Notepad ++ blinks when clicking different lines

### ISSUE THREAD

**Issue by marat569** [notepad-plus-plus/notepad-plus-plus#15716]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description of the Issue

Honestly I dont know how to better explain it.

When I click different lines, notepad++ starts blinking the lines I clicked

I uninstalled np++, deleted the folder in appdata -- idk whats wrong

Video: https://streamable.com/9e4iq9

### Steps To Reproduce

1.  open a file (ini file in my case)
2.  start clicking different lines

### Current Behavior

stuff blinks

### Expected Behavior

stuff doesn't blink

### Debug Information

```shell
Notepad++ v8.7   (64-bit)
Build time : Sep 17 2024 - 17:06:31
Path : C:\Program Files\Notepad++\notepad++.exe
Command Line : "C:\Program Files\Special K\Profiles\TEKKEN 8\SpecialK.ini"
Admin mode : OFF
Local Conf mode : OFF
Cloud Config : OFF
Periodic Backup : ON
OS Name : Windows 11 Pro (64-bit)
OS Version : 23H2
OS Build : 22631.4249
Current ANSI codepage : 1252
Plugins : 
    mimeTools (3.1)
    NppConverter (4.6)
    NppExport (0.4)
```

**Comment by xomx** [comment]
Cannot reproduce.

This looks to me as a graphics card drivers problem, but it can also be affected by another installed SW (e.g. in the past the N++ had graphics conflicts when Slack was running). You can try updating the drivers or if your PC has dual graphics (e.g. integrated Intel and discrete Nvidia), check what happens if the N++ will be running with the other card active than now, e.g.:

![npp-nvidia-intel](https://github.com/user-attachments/assets/6021d2a4-d385-4193-bc0a-e15664964f1b)

**Comment by marat569** [comment]
Also I forgot to add its only the .ini preset that does this

if I open the same file, and change the language to literally anything else; no issues

**Comment by marat569** [comment]
I also added the ini files I'm having trouble with, they're generated by a tool I use

Maybe the ini highlighting is conflicting with this somehow

[SpecialK.zip](https://github.com/user-attachments/files/17452049/SpecialK.zip)

**Comment by xomx** [comment]
> its only the .ini preset that does this

Yes, as I saw the INI format in the video, I tried also with the "C:\\Windows\\win.ini" sample and it was ok.
Your SpecialK file is ok too for me.

Can you try the same with a portable Notepad++ version?
- dl: https://github.com/notepad-plus-plus/notepad-plus-plus/releases/download/v8.7/npp.8.7.portable.x64.zip
- unzip to some dir, then run there the notepad++.exe and try to reproduce your issue

**Comment by marat569** [comment]
Some more news -- I downgraded to np++ 8.4.7 and there is no issue -- so this is something that happened recently

**Comment by marat569** [comment]
> > its only the .ini preset that does this
> 
> Yes, as I saw the INI format in the video, I tried also with the "C:\Windows\win.ini" sample and it was ok. Your SpecialK file is ok too for me.
> 
> Can you try the same with a portable Notepad++ version?
> 
> * dl: https://github.com/notepad-plus-plus/notepad-plus-plus/releases/download/v8.7/npp.8.7.portable.x64.zip
> * unzip to some dir, then run there the notepad++.exe and try to reproduce your issue

the portable version seems to work -- does np++ install to any regkeys? I delete the `\AppData\Roaming\Notepad++` file
